.env.example

```diff
@@ -12,7 +12,6 @@
 # NMDC_MONGO_USER=changeme
 # NMDC_MONGO_PASSWORD=changeme
 
-NMDC_HOST=http://127.0.0.1:8080
```

Comment:
Should this also be `8000`?

Reply:
Yes, `8000` is correct, but that line has been removed because it is unnecessary now that there is a default value set in `config.py`.